contracts/AaveV3Lender.sol

```diff
@@ -242,9 +253,11 @@ contract AaveV3Lender is BaseStrategy, UniswapV3Swapper, AuctionSwapper {
     }
 
     function checkCooldown() public view returns (bool) {
-        uint256 cooldownStartTimestamp = IStakedAave(stkAave).stakersCooldowns(
-            address(this)
-        );
+        if (block.chainid != 1) return false;
+
+        uint256 cooldownStartTimestamp = IStakedAave(stkAave)
+            .stakersCooldowns(address(this))
+            .timestamp;
```

Comment:
Instead of using `COOLDOWN_SECONDS()` (deprecated) use the new `getCooldownSeconds()` function.

https://etherscan.io/address/0x0fe58fe1caa69951dc924a8c222be19013b89476#code#F14#L87